### DISCUSSION TITLE
Funktion

### DISCUSSION THREAD

**Discussion by Huckumer** [2Abendsegler/GClh#2667] [Questions] [answered]
.

**Comment by 2Abendsegler** [comment]
Hallo,
hat sich die ursprüngliche Frage erledigt oder benötigst du noch Tipps?
LG
Frank

**Comment by Huckumer** [comment, marked as answer]
nAbend, hat sich erledigt, danke der Nachfrage. Hatte versucht die Frage zu löschen, geht aber wohl nicht. Gruß, Uwe. --Gesendet mit der GMX Mail App
